modules/ROOT/pages/tutorial-manage-an-api.adoc

```diff
@@ -82,6 +85,15 @@ image::rlp-conditions.png[]
 +
 If a RAML spec is attached to your API, click *Preview resource matching* to verify which resources your filters affect.
 
+== Next Steps
+* Register a client application to access the API with the newly applied policy:
+. xref:exchange::to-request-access.adoc[Request Access from Exchange]
```

Comment:
If these are steps rather than a bullet list of xrefs, then reformat the xrefs as sentences:
```suggestion
. xref:exchange::to-request-access.adoc[Request access from Exchange.]
```
Ditto below.